reference/configuration/framework.rst

```diff
@@ -1093,8 +1093,61 @@ version_strategy
 
 **type**: ``string`` **default**: ``null``
 
-The service id of the :doc:`asset version strategy </frontend/custom_version_strategy>`
-applied to the assets.
+The service id of the asset version strategy applied to the assets. In addition
+to your own :doc:`custom asset version strategies </frontend/custom_version_strategy>`,
+Symfony defines two services for its :ref:`built-in strategies <component-assets-versioning>`:
+``assets.empty_version_strategy`` and ``assets.static_version_strategy``.
+
+The ``assets.empty_version_strategy`` is useful to override the global asset
+versioning strategy inside an asset package:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+       framework:
+            assets:
+                # this strategy is applied to every asset (including packages) ...
```

Comment:
this strategy is applied to all assets of the default package, and to all custom packages, *unless they configure their own version or their own version strategy*